numpy/ma/core.py

```diff
@@ -1489,7 +1489,7 @@ def is_mask(m):
 
     See Also
     --------
-    isMaskedArray : Test whether input is an instance of MaskedArray.
+    ma.isMaskedArray : Test whether input is an instance of MaskedArray.
```

Comment:
`is_mask` is on the page [`routines.ma`](https://16783-908607-gh.circle-artifacts.com/0/doc/build/html/reference/routines.ma.html) (I think this is because `is_mask` is a function in `np.ma`, not a method of the class `MaskedArray`). And we might have to insert `isMaskedArray` (also `isarray`, `isMA`) right after `is_mask`.